docs/CLI.md

```diff
@@ -25,7 +25,7 @@ COMMANDS:
 
 GLOBAL OPTIONS:
    --help, -h     show help (default: false)
-   --version, -v  print the version (default: false)
+   --version, -v  print the version (default: false)`
```

Comment:
It's weird that this extra backtick is coming up here, but it seems like a bug upstream and not in our code.

Reply:
This looks like my bad in copying, let me remove

Reply:
fixed in [d6a2dd8](https://github.com/sourcegraph/scip/pull/148/commits/d6a2dd884e8323c90ca684a9caef6d775506edb2)